### ISSUE TITLE
Django restfull docker

### ISSUE THREAD

**Issue by jduckles** [ouinformatics/ccstack-docker#1]
Should we try to go to nginx?

**Comment by mbstacy** [comment]
Yes, nginx seems to be the right web server. If we fail, can always go back to apache

**Comment by jduckles** [comment]
Or http://gunicorn.org/

**Comment by jduckles** [comment]
Or perhaps them combined together, so that static assets can also be served. This may be a good place to start from: https://github.com/rehabstudio/docker-gunicorn-nginx

**Comment by mbstacy** [comment]
I was just going to add we should run together. http://docs.gunicorn.org/en/latest/deploy.html

**Comment by jduckles** [comment]
:+1: together it is.
